src/gui-wizard-gtk/wizard.c

```diff
@@ -2494,9 +2495,9 @@ static void on_page_prepare(GtkNotebook *assistant, GtkWidget *page, gpointer us
 
     if (pages[PAGENO_EVENT_PROGRESS].page_widget == page)
     {
-        log_info("g_event_selected:'%s'", g_event_selected);
         if (g_event_selected && g_event_selected[0])
         {
+            log_notice("Selected event '%s'; going to run it now...", g_event_selected);
```

Comment:
I don’t know, it feels to me like these messages should rather be logged under the debug level.

Reply:
Yeah, the logging levels are all over the place. I'd place most of the new messages on info at least, although we don't really differentiate between info and debug.

I was mostly using notice here, because I wanted to see the messages, but no bother with all the other, lower-level noise.

Reply:
Oh, nevermind, I didn’t notice that `LOG_NOTICE` requires `-v`. Yeah, then probably that’s fine.

Reply:
RESOLVED WHOCARES